.github/workflows/ci.yml

```diff
@@ -114,73 +111,36 @@ jobs:
             ${{ runner.os }}-go-build-
       - name: Prepare
         id: prep
+        env:
+          GITHUB_EVENT_NAME: ${{ github.event_name }}
```

Comment:
`GITHUB_EVENT_NAME` should already exist: [Default Environment Variables](https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables)

Reply:
Removed